File: 이훈일/Programmers/Level2/이진 변환 반복하기.java
class Solution {
    public int[] solution(String s) {
        int[] result = new int[2];
        while(!s.equals("1")) {
            String rep = s.replaceAll("0", "");
            result[1] += s.length() - rep.length();
            s = Integer.toString(rep.length(), 2);
            result[0]++;
        }
        
        return result;
    }
}

File: 조준장/level2/이진 변환 반복하기.kt
class Solution {
    fun solution(s: String): IntArray {
        var answer: IntArray = IntArray(2)
        var str : String = s
        
        while (str != "1"){
            answer[0]++
            answer[1] += str.replace("1","").length
            str = str.replace("0","").length.toString(2)
            
        }
        
        return answer
    }
}
